tools/sigh.js

```diff
@@ -605,6 +613,10 @@ function importSpotify(args) {
 
 // Watches for file changes, then runs the `arg` steps.
 function watch([arg, ...moreArgs]) {
+  if (chokidar == null) {
+    console.log('\nthe sigh watch subcommand requires chokidar to be installed. Please run \'npm install chokidar\' then try again\n');
```

Comment:
`npm install` will add chokidar to prod dependencies (default behavior). We should probably recommend `--no-save` here.

Reply:
hmm, that's .. new. OK I'll update the printout.